pallets/detach/src/lib.rs

```diff
@@ -33,6 +36,7 @@ pub mod crypto {
 		traits::Verify,
 		MultiSignature, MultiSigner,
 	};
+	/// ¿
```

Comment:
This is some internal thing of substrate, it's fine to not document it on our codebase actually